### DISCUSSION TITLE
Search mode: how to move left/right in the search text field

### DISCUSSION THREAD

**Discussion by eugenesvk** [wezterm/wezterm#3086] [Q&A] [answered]
Let's say I've enabled the search mode, typed `pastte`, realized I've made a typo and would like to move 1 char left and delete the extra `t` (or typed `pase` and want to move 1 char left to insert the missing `t`)
How do I do that? ◀▶ don't seem to work
I've tried to bind them to `'MoveLeft'`,`'MoveRight'` via `action=act.CopyMode`, but that moves the cursor around the selected text item, not in the search text field itself

**Comment by wez** [comment, marked as answer]
That is not currently supported

**Reply by eugenesvk** [reply]
Thanks, I've added a feature request https://github.com/wez/wezterm/issues/3087
